src/App/Model/CryptAES.php

```diff
@@ -37,7 +37,7 @@ public function decrypt($crypt)
     {
         if (!$this->cipher && !$this->setCipher()) return $crypt;
 
-        return $this->cipher->decrypt($crypt);
+        return $this->cipher->decrypt(base64_decode($crypt));
```

Comment:
Here is the error-message: 

`General error: 1366 Incorrect string value: '\xF7\xEElVVC...' for column 'username' at row 1`